build.sh

```diff
@@ -806,22 +899,26 @@ case "$COMMAND" in
         exit 0
         ;;
 
-    "examples-ios")
+    "examples-ios")     
         workspace="examples/ios/objc/RealmExamples.xcworkspace"
+
+        # Build Realm to be use in the 'RealmExamples' project
+        xc -scheme Realm -configuration "$CONFIGURATION" -sdk iphonesimulator -workspace "$workspace"
```

Comment:
This shouldn't be necessary or useful? The example should automatically build its dependencies when needed.